tests/unit/lms/models/application_settings_test.py

```diff
@@ -43,6 +44,17 @@ def test_set(self, application_settings, group, key, value, expected_value):
 
         assert application_settings[group][key] == expected_value
 
+    def test_secrets(self, application_settings, aes_service):
```

Comment:
I reckon for this type of functionality that deals with values (that need to be base64 -> encoded) this type of test is preferable to mocking aes_service, base64, etc.

Other tests in the module are similar too.